### DISCUSSION TITLE
New codename for Node v22!

### DISCUSSION THREAD

**Discussion by ryuujo1573** [nodejs/Release#1031] [General]
It goes with letter `J`, why don't we pick <kbd>Java</kbd> as the brand new code name!

Most likely this will ring the bell in the world 🌋

Sorry for any interruption this conversation can be🤣

**Comment by ljharb** [comment]
it's already `Jod`, as you can see in https://github.com/nodejs/Release/blob/main/CODENAMES.md

**Reply by ryuujo1573** [reply]
Thank you for your reply!

I see "Codenames for future releases are subject to change" so maybe there's a chance to change or...<del>I'm calling it <kbd>java</kbd> on my behalf 😂 </del>
